bin/base/__init__.py

```diff
@@ -63,5 +64,8 @@ def __init__(self, label, section_arguments=()):
         'Computation options',
         ('output', 'max_time_step', 'total_time')
     ),
-    Section('Advanced', ())  # TODO: Add ('preparation_only',))
+    Section(
```

Comment:
QGIS should also add exported temp layers into map canvas.

Reply:
See https://github.com/storm-fsv-cvut/smoderp2d/issues/228#issuecomment-1762757992

Reply:
Done in [cc90cf1](https://github.com/storm-fsv-cvut/smoderp2d/pull/284/commits/cc90cf12c163dfab85b917ebb1059715b392cbaf). Thanks bro.